user_modules/shared_testcases/commonFunctions.lua

```diff
@@ -720,4 +722,55 @@ function commonFunctions:SDLForceStop(self)
 	os.execute("sleep 1")
 end
 
+---------------------------------------------------------------------------------------------
+--14. Function gets parameter from ini file
+---------------------------------------------------------------------------------------------
+function commonFunctions:read_parameter_from_ini(param_name)
+	local sdl_ini_file_name = config.pathToSDL .. "smartDeviceLink.ini"
+
+	local file = assert(io.open(sdl_ini_file_name, "rb"))
+	if not file then
+		return nil
+	end
+	local param_value  = nil
+	for line in io.lines(sdl_ini_file_name) do
+		if string.sub(line, 1 , 1) ~= ";" then
+			if string.find(line, param_name.."%s") ~= nil then
+				local b, e = string.find(line, "%s".."=".."%s.")
+				if b ~= nil then
+					local len = string.len(line)
+					param_value = string.sub(line, e, len)
+					break
+				end
+			end
+		end
+	end
+	file:close()
+	return param_value
```

Comment:
@istoimenova Kovalenko I required this. If ini file doesn't contain parameter with name,  method returns nil. I think you can check this value before using.